src/lib/tableland-calls.ts

```diff
@@ -31,38 +36,49 @@ async function sendResponse(res: any) {
   return json.result;
 }
 
+// Take an Object with any symantic for key naming and return a new Object with keys that are lowerCamelCase
+// Example: `camelCaseKeys({structure_hash: "123"})` returns `{structureHash: "123"}`
+function camelCaseKeys(obj: object) {
+  return Object.fromEntries(
+    Object.entries(obj).map((entry: KeyVal) => {
+      const key = entry[0];
+      const val = entry[1];
+      return [camelCase(key), val];
+    })
+  );
+}
+
 async function GeneralizedRPC(
   this: Connection,
   method: string,
-  statement: string,
-  tableId?: string,
-  options?: any
+  params: RpcParams = {}
 ) {
-  const signer = this.signer;
-  const address = await signer.getAddress();
-
-  const params = [
-    {
-      statement: statement,
-      id: tableId,
-      controller: address,
-      type: options?.type,
```

Comment:
As far as I can tell`type` is not being used.  Maybe an artifact from an older RPC spec?

Reply:
This is correct.

Reply:
Yeh nice catch, I think you're right.